src/main/resources/application-dev.yml

```
@@ -113,4 +113,4 @@ supabase:
 
 # CORS Configuration
 cors:
-  allowed-origins: ${CORS_ALLOWED_ORIGINS:http://localhost:3000,https://twiggle.tech,https://*.vercel.app}
\ No newline at end of file
+  allowed-origins: ${CORS_ALLOWED_ORIGINS:http://localhost:3000,https://*.github.dev,https://twiggle.tech,https://*.vercel.app}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)